### ISSUE TITLE
Handle Unicode filenames in forward search

### ISSUE THREAD

**Issue by igorlfs** [latex-lsp/texlab#854]
Hello!

I'm using texlab with neovim + latexmk + zathura and I noticed that I can't use forward search when a document contains a non-ascii character (e.g., é.tex). I can build such files just fine, but when I try to forward search, the following error is logged:

```
ERROR - Forward search failed: PDF document '/path/to/pdf/%c3%a9.pdf' not found
```

Oddly, the tex file doesn't error. I also tried hard-coding the path to a sample PDF, and it still didn't work. Not sure if it's an issue downstream, but this felt like the right place to start.

**Comment by pfoerster** [comment]
@igorlfs It looks like this is a duplicate of #848, which should be fixed with 178cba7eeb51a03175e101bffbb25f7090705311.

**Comment by igorlfs** [comment]
Oh, my bad! I Should have made a more throughout search before creating a new issue 😅
Anyways, thanks for the fix!
